xls/tests/documentation_test.py

```diff
@@ -122,6 +124,29 @@ def test_dslx_blocks(self, i: int, dslx_block: str) -> None:
       print(p.stderr)
     self.assertEqual(p.returncode, 0)
 
+  @parameterized.named_parameters(get_examples())
+  def test_dslx_blocks_fmt(self, i: int, dslx_block: str) -> None:
+    """Checks a DSLX block is canonically formatted."""
+    example = strip_attributes(dslx_block)
+    input = example.dslx.lstrip()
```

Comment:
please dont redefine builtin.

I'd hoped the pylintrc I'd added would catch stuff like this :(

Reply:
Done.